example-apps/Token-EasySending/src/web3/account.ts

```diff
@@ -21,6 +21,14 @@ async function isContract(provider: Provider, address: string) : Promise<boolean
     return false;
 }
 
+export function createNameStruct(input: string, type: "email" | "twitter"): NameStruct {
```

Comment:
If we can use getName() fn which is defined above to fetch the user corresponding NameStruct, maybe we do not need this one to create a new Struct by passing in value